common-js/node_modules/.bin/_mocha

```diff
@@ -0,0 +1 @@
+../mocha/bin/_mocha
```

Comment:
Make sure `node_modules/` is in your .gitignore. We shouldn't be pushing these binaries